src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -19,7 +19,7 @@
 #include <openrct2/localisation/Localisation.h>
 #include <openrct2/management/Research.h>
 #include <openrct2/network/network.h>
-#include <openrct2/object/ObjectList.h>
+#include <openrct2/object/ObjectList.h>  // Unused?
```

Comment:
This is not related to your PR, so please revert it.

Also, this kind of comment is not very useful: if there is any doubt a header is used, it should be checked and removed if it isn't used.

Reply:
Deleted Comment